src/browser/css/css.zig

```diff
@@ -23,6 +23,18 @@ const std = @import("std");
 const Selector = @import("selector.zig").Selector;
 const parser = @import("parser.zig");
 
+pub const Interfaces = .{
+    CSS,
+};
+
+// https://developer.mozilla.org/en-US/docs/Web/API/CSS
+pub const CSS = struct {
+    pub fn _supports(_: []const u8, _: ?[]const u8) bool {
```

Comment:
I know it's doing nothing, but since this is currently the only code that's using the new `checkSelfReceiver` and the subsequent flow, I think a test, as trivial as it might seem, is needed:


```zig
const testing = @import("../../testing.zig");
test "Browser.HTML.CSS" {
    var runner = try testing.jsRunner(testing.tracking_allocator, .{});
    defer runner.deinit();

    try runner.testCases(&.{
        .{ "CSS.supports('over 9000')", "true" },
    }, .{});
}
```